resources/todomvc/architecture-examples/lit/package.json

```diff
@@ -1,6 +1,10 @@
 {
     "private": true,
     "type": "module",
+    "engines": {
+        "node": ">=18.13.0",
+        "npm": ">=8.19.3"
+    },
```

Comment:
you need to run `npm i` and commit package-lock.json again after this change :-)

Reply:
Argh, I always forget! Thanks for catching that